test/test/boot.js

```diff
@@ -449,6 +449,37 @@ async function checkSupport() {
   }
 }
 
+function checkClearKeySupport() {
+  const clearKeySupport = shakaSupport.drm['org.w3.clearkey'];
+  if (!clearKeySupport) {
+    return false;
+  }
+  return clearKeySupport.encryptionSchemes.includes('cenc');
+}
+
+function checkPlayReadySupport() {
+  return shakaSupport.drm['com.microsoft.playready'] ||
+      shakaSupport.drm['com.microsoft.playready.recommendation'];
```

Comment:
Add `com.chromecast.playready` and other variations

Reply:
PlayReady on chromecast is deprecated an no longer available on new devices